go.mod

```diff
@@ -0,0 +1,63 @@
+module github.com/Stackdriver/stackdriver-prometheus-sidecar
+
+require (
+	cloud.google.com/go v0.43.0
+	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
+	contrib.go.opencensus.io/exporter/prometheus v0.1.0
+	contrib.go.opencensus.io/exporter/stackdriver v0.12.4
+	github.com/Azure/azure-sdk-for-go v32.1.0+incompatible // indirect
+	github.com/Azure/go-autorest v12.4.1+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.5.0 // indirect
+	github.com/Azure/go-autorest/autorest/to v0.2.0 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.1.0 // indirect
+	github.com/Azure/go-autorest/tracing v0.2.0 // indirect
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4 // indirect
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878 // indirect
+	github.com/aws/aws-sdk-go v1.21.10 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-kit/kit v0.9.0
+	github.com/gogo/protobuf v1.2.2-0.20190730201129-28a6bbf47e48 // indirect
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/golang/protobuf v1.3.2
+	github.com/googleapis/gnostic v0.3.0 // indirect
+	github.com/gophercloud/gophercloud v0.3.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/grpc-ecosystem/grpc-gateway v1.9.5 // indirect
+	github.com/hashicorp/go-immutable-radix v1.1.0 // indirect
+	github.com/hashicorp/go-msgpack v0.5.5 // indirect
+	github.com/hashicorp/go-rootcerts v1.0.1 // indirect
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/hashicorp/memberlist v0.1.4 // indirect
+	github.com/hashicorp/serf v0.8.3 // indirect
+	github.com/jpillora/backoff v0.0.0-20180909062703-3050d21c67d7 // indirect
+	github.com/json-iterator/go v1.1.7 // indirect
+	github.com/miekg/dns v1.1.15 // indirect
+	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
+	github.com/oklog/oklog v0.3.2
+	github.com/onsi/ginkgo v1.8.0 // indirect
+	github.com/onsi/gomega v1.5.0 // indirect
+	github.com/opentracing/opentracing-go v1.1.0 // indirect
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/common v0.4.1
+	github.com/prometheus/procfs v0.0.3 // indirect
+	github.com/prometheus/prometheus v0.0.0-20190710134608-e5b22494857d
```

Comment:
what's your opinion on using stable version of prometheus, e.g. `2.11.1`?

Reply:
I believe that's already what we're using. This file was updated by `go get github.com/prometheus/prometheus@v2.11.1`, and running that command again isn't making any changes to this file.

Reply:
Found that the version may be recorded as pseudo-version. The details are described in [Modules FAQ](https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-package-that-has-not-opted-in-to-modules). I verified that the commit hash `e5b22494857d` matches the prometheus version `v2.11.1`.